UITests/Common/BookmarksUtilites.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  BookmarksUtilites.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+
+class BookmarkUtilities {
```

Comment:
💡 'I’m having second thoughts about this class. I think another approach is to have an extension of `XCUIApplication` and define those methods there. Thoughts?

Then it’s easier to read in the tests:

```swift
app.openBookmarksManager(..)
app.openSiteToBookmark(…)
app.dismissPopover(…)
// etc ...
```

Reply:
Example here: https://github.com/duckduckgo/macos-browser/pull/3162/files#diff-3b229a7e4f5bfeae7c04d751aeb24bc7c35282ce32266902ab2e24a4663002cc

Reply:
I second your idea 👍 would much better prefer to have an extension on XCUIApplication than standalone utility classes.

Reply:
Great. I’ve done that ✅